transform/mattermost-analytics/models/marts/product/dim_server_info.sql

```diff
@@ -0,0 +1,35 @@
+with hosting_type_info as (
+    -- Summarize server hosting type information
+    select
+        server_id,
+        -- Aggregate is cloud in order to validate whether the server is cloud or not
+        count_if(is_cloud = true) as count_is_cloud_days,
+        count_if(is_cloud = false) as count_not_is_cloud_days,
+        count_if(is_cloud is null) as count_unknown_is_cloud_days
+    from
+         {{ ref('int_server_active_days_spined') }}
+    group by
+        server_id
+), latest_values as (
+    -- Get latest values for installation id (if exists) and full version string
+    select
+        server_id,
+        installation_id
+    from
+         {{ ref('int_server_active_days_spined') }}
+    -- Keep latest record per day
+    qualify row_number() over (partition by server_id order by activity_date desc) = 1
```

Comment:
instead of latest record, if we use the last value where it was `not null` we can get the `installation_id`, since this is a dimension table.

Reply:
The idea of this CTE is to keep the latest installation id in case the server reports multiple installation ids. There are a few cases that this happens.